src/templates/our-story.twig

```diff
@@ -16,17 +16,32 @@ cafe in the heart of local community, where speciality coffees are savoured in a
         GHC offers a full spectrum of brewing opportunities—from pour overs to syphon pots to finely executed
         espresso beverages. Each cup of coffee is made with love, precision and care.
     </p>
-    <img src="/img/espresso-machine_jya3mn_c_scale,w_443.jpg"
+    <img src="/img/espresso-machine_v89s2i_c_scale,w_320.jpg 320w"
```

Comment:
320w shouldn't be here

Reply:
Also 320px is not wide enough i'd use 547 or 730 as a fallback size for old browsers